### ISSUE TITLE
loss function in mario_curio.py

### ISSUE THREAD

**Issue by osm040836** [jcwleo/mario_rl#25]
안녕하세요, mario_curio.py 파일에서 의문점이 들어 issue 남깁니다.
Line 277번째 줄에서 다음과 같이 entropy loss 항이 추가되어 계산되어야 하지 않나요? 

기존)  loss = (actor_loss + 0.5 * critic_loss) + icm_scale * \
                    ((1 - beta) * inverse_loss + beta * forward_loss)

변경)  loss = (actor_loss + 0.5 * critic_loss - entropy_coef * entropy) + icm_scale * \
                ((1 - beta) * inverse_loss + beta * forward_loss)'

감사합니다.
